packages/node_modules/pouchdb-validation/README.md

```diff
@@ -0,0 +1,10 @@
+pouchdb-validation
+==================
+
+A PouchDB plug-in that allows you to re-use your CouchDB
+validate_doc_update functions on the client side. A browser version is
+available.
+
+See also [pouchdb-validation's documentation](http://pythonhosted.org/Python-PouchDB/js-plugins.html#pouchdb-validation-plug-in)
```

Comment:
I plan to do a post-monorepo-merge README cleanup, there are several things that need to be done